.gitignore

```
@@ -4,8 +4,8 @@ target/
 !**/src/test/**/target/
 
 ### IntelliJ IDEA ###
-.idea/modules.xml
-.idea/jarRepositories.xml
+.idea*
+.idea
```

nit: I think `.idea*` is enough here 🤔 
```suggestion
.idea*
```